File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/ParentsComparison/ParentsComparisonPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Concepts.ClientAPI.ParentsComparison
{
    public partial class ParentsComparisonPage : UserControl
    {
        public ParentsComparisonPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the Click event of the objButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void objButton_Click(object sender, EventArgs e)
        {
            VWGClientContext.Current.Invoke("compareParents");
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/ClientAPI/ParentsComparison/ParentsComparisonPage.vb
Namespace CompanionKit.Concepts.ClientAPI.ParentsComparison

	Public Class ParentsComparisonPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles the Click event of the objButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub objButton_Click(sender As System.Object, e As System.EventArgs) Handles objButton.Click
            VWGClientContext.Current.Invoke("compareParents")
        End Sub
    End Class

End Namespace